cpp/velox/benchmarks/QueryBenchmark.cc

```diff
@@ -70,7 +73,11 @@ auto bm = [](::benchmark::State& state,
   std::vector<std::shared_ptr<velox::substrait::SplitInfo>> scanInfos;
   scanInfos.reserve(datasetPaths.size());
   for (const auto& datasetPath : datasetPaths) {
-    scanInfos.emplace_back(getSplitInfos(datasetPath, fileFormat));
+    if (datasetPath.back() == '/') {
```

Comment:
Please use `std::filesystem::is_directory()` to check if it is a directory, directory path may not ends with '/'